### DISCUSSION TITLE
Wann Anpassung Schema Impfzertifikat an neuen Impfstoff Nuvaxovid von Novavax?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#216] [Q&A] [answered]
Hallo und ein frohes neues Jahr!

Bundesgesundheitsminister Karl Lauterbach gab Ende Dezember im Rahmen einer Pressekonferenz bekannt, dass mit einer ersten Lieferung des proteinbasierten Impfstoffs Nuvaxovid von Novavax bereits im Januar 2022 zu rechnen sei.

Wann wird das Schema für das Impfzertifikat entsprechend angepasst?

Nuvaxovid von Novavax (NVX-CoV2373, EU-Zulassungsnummer EU/1/21/1618) 

Insbesondere ist unklar welcher Wert für das Feld 'vp'  (vaccine prophylaxis as defined by SNOMED CT)
verwendet werden soll (eine Ausprägung für "Totimpfstoff bzw. rekombinanter Proteinimpstoff"). 

Die anderen werte sind vermutlich:
mp = EU/1/21/1618      (medicinal product used for this specific dose of vaccination)
ma = ORG-100032020  (Marketing authorisation holder or manufacturer,)

Bis die PVS-Updates in den Praxen installiert sind braucht ja auch ein wenig Vorlauf...

**Comment by ChristianFischerJena** [comment]
Hallo liebe Api-Entwickler, wir würden uns ebenfalls sehr über eine Antwort bzw. Anpassung der Api freuen. Der neue Impfstoff wird in Kürze verimpft werden und es wäre toll, wenn die Zertifikatsausstellung für diese Impfungen bis dahin möglich wäre. Vielen Dank!

**Comment by MoritzIBM** [comment, marked as answer]
Liebe Kolleg*innen aus Kiel und Jena,

das Schema wird zeitnah (i.e., ca. in den nächsten 3 Wochen, aber garantiert vor dem Impfstart) angepasst.
Nuvaxovid 
mp=EU/1/21/1618 
ma=ORG-100032020 
vp=J07BX03 
(vorher bekannt als nVX-CoV2373.)

Ich hoffe das hilft weiter. Beste Grüße!
Moritz

**Reply by kamianni** [reply]
Ist die Bezeichnung Novaxovid richtig? Sollte es nicht Nuvaxovid sein?

**Reply by MoritzIBM** [reply]
Oh ja, danke - ich habe es korrigiert!

**Reply by mge-medisoftware** [reply]
In RU funktioniert das nun mit Nuvaxovid. Vielen Dank!

**Reply by mge-medisoftware** [reply]
Ist nun auch gut beschrieben [Implementierungsleitfaden](https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/Implementation.md)

**Comment by ChristianFischerJena** [comment]
Vielen Dank für Ihre Antwort. Das hilft schon einmal weiter! Können Sie auch schon sagen, welcher Wert im Feld vp erwartet werden wird?

Verzeihen Sie meine Ungeduld. Wir (und sicherlich auch andere Systeme) brauchen halt immer etwas Vorlauf, um die Änderung dann auszurollen. Daher sind wir für jeden Tag zusätzlichen Puffer dankbar.

Beste Grüße

**Reply by kamianni** [reply]
Hallo,

MoritzIBM hat die Antwort ein wenig hinter "ma" versteckt.

vp=J07BX03 (Bezeichnung: covid19-vaccines)

Viele Grüße

**Reply by mge-medisoftware** [reply]
Hallo,

hmmmmm, der ATC J07BX03 passt aber  eher nicht zu SNOMED-CT Kodierung bisheriger 'vp'-Werte wie 1119349007=Vaccine product containing only Severe acute respiratory syndrome coronavirus 2 messenger ribonucleic acid (medicinal product)

Viele Grüße aus Kiel

**Reply by MoritzIBM** [reply]
Hallo zusammen,

für Comirnaty und Spikevax ist es "1119349007 or J07BX03". Für Janssen und Vaxzevria ist der vaccine type nur J07BX03, das gleiche gilt für Nuvaxovid.

Beste Grüße!

**Reply by mge-medisoftware** [reply]
Hallo nochmal, 

wir verwenden schon seit "Anbeginn" für für Janssen und Vaxzevria "1119305005", ist das demnächst dann nicht mehr zulässig?

Danke und viele Grüße

**Reply by MoritzIBM** [reply]
Hallo zusammen,

ja, das sollte laut neuer EU Spec ( Value Set (12/01) geändert werden:
https://ec.europa.eu/health/system/files/2022-01/digital-green-value-sets_en.pdf

Beste Grüße und schönes Wochenende!

**Reply by ChristianFischerJena** [reply]
Das scheint zu klappen. Vielen Dank dafür!
